### ISSUE TITLE
GitLab should not notify an unsuccessful build if allow_failure is enabled

### ISSUE THREAD

**Issue by erickskrauch** [Requilence/integram#40]
Hi. We use your tool and that's cool, but for one of the projects we have this config:

```yml
test_jre7:
  <<: *test_definition
  image: openjdk:7-jdk-alpine

test_jre8:
  <<: *test_definition
  image: openjdk:8-jdk-alpine

# Allow failure, until https://github.com/docker-library/openjdk/issues/101#issuecomment-312510546
test_jre9:
  <<: *test_definition
  image: openjdk:9-jdk
  allow_failure: true
```

As you can see, task for `jre9` have `allow_failure: true`, because some Java 9 bug (not ours, honestly). But the bot continues to send a notification that this task has been failed:

![](http://erickskrauch.ely.by/monosnap-26-07-2017-at-14-24-19.png)

If this is possible, please, add an option or enable it by default to do not receive such notifications for suppressed tasks.

**Comment by Requilence** [comment]
HI! Thanks for reporting. Should be fixed by #9741c2d62142505503657f10d2273b9aad29a83d. Already on production

**Comment by erickskrauch** [comment]
Thanks! It's worked:

![](http://erickskrauch.ely.by/monosnap-26-07-2017-at-15-30-32.png)
